nomos-services/data-availability/dispersal/src/backend/kzgrs.rs

```diff
@@ -1,20 +1,23 @@
-use crate::adapters::mempool::DaMempoolAdapter;
-use crate::adapters::network::DispersalNetworkAdapter;
-use crate::backend::DispersalBackend;
+// std
+use std::sync::Arc;
+use std::time::Duration;
+// crates
 use futures::StreamExt;
 use itertools::izip;
 use kzgrs_backend::common::blob::DaBlob;
 use kzgrs_backend::common::{build_blob_id, Column, ColumnIndex};
-use kzgrs_backend::encoder;
 use kzgrs_backend::encoder::EncodedData;
+use kzgrs_backend::{dispersal, encoder};
 use nomos_core::da::{BlobId, DaDispersal, DaEncoder};
 use overwatch_rs::DynError;
```

Comment:
nitpick: also internal (?)

Reply:
Up until now I considered internal modules to be `crate` and `super`. I'll fix it, just want to confirm.

Reply:
yeah, maybe I should have clarified on that. internal would be anything we own.